docs/main/usage_cli/common_examples/settings_examples/ex3.rst

```diff
@@ -0,0 +1,111 @@
+
+.. code:: shell
+
+    $ axonshell system settings-global --help
+
+    Usage: axonshell system settings-global [OPTIONS] COMMAND [ARGS]...
+
+      Group: Global Settings.
+
+    Options:
+      --help  Show this message and exit.
+
+    Commands:
+      configure-destroy             Enable or disable dangerous API endpoints.
+      get                           Get all settings.
+      get-section                   Get settings for a section.
+      get-sub-section               Get settings for a subsection.
+      update-section                Update a section from arguments.
+      update-section-from-json      Update a section from a JSON file.
+      update-sub-section            Update a subsection from arguments.
+      update-sub-section-from-json  Update a subsection from a JSON file.
+
+If you are looking to update a section or sub-section using only command line parameters, one of the commands
+you could use would be the ``update-sub-section`` command.
+
+.. code:: shell
+
+    $ axonshell system settings-global update-sub-section --help
+
+    Usage: axonshell system settings-global update-sub-section
+               [OPTIONS]
+
+      Update a subsection from arguments.
+
+    Options:
+      -u, --url URL                   URL of an Axonius instance  [env var:
+                                      AX_URL; required]
+      -k, --key KEY                   API Key (or username if credentials=True) of
+                                      user in an Axonius instance  [env var:
+                                      AX_KEY; required]
+      -s, --secret SECRET             API Secret (or password if credentials=True)
+                                      of user in an Axonius instance  [env var:
+                                      AX_SECRET; required]
+      -xf, --export-format [json-full|json-config|str]
+                                      Format of to export data in  [env var:
+                                      AX_EXPORT_FORMAT; default: str]
+      -se, --section TEXT             Settings section internal name (not title)
+                                      [env var: AX_SECTION; required]
+      -sb, --sub-section TEXT         Settings sub section internal name (not
+                                      title)  [env var: AX_SUB_SECTION; required]
+      -c, --config SPLIT_EQUALS       Configuration keys in the form of key=value
+                                      (multiples)  [env var: AX_CONFIG]
+      --help                          Show this message and exit.
+
+An example might look like this:
+
+.. code:: shell
+
+    $ axonshell system settings-global update-sub-section --section "email_settings" \
+    --config "sender_address=no-reply@yourdomain.com"
+
+To update a section or sub-section using json from a file you could use the ``update-sub-section-json`` command.
+
+.. code:: shell
+
+    $ axonshell system settings-global update-sub-section-from-json --help
```

Comment:
@nate-axonius is there an example to how the json file looks like?

Reply:
Fixed.